docs/flow-visibility.md

```diff
@@ -0,0 +1,45 @@
+#Flow Visibility in Antrea
+
+[Antrea](architecture.md) implements Container Network Interface(CNI) and Network Policy (both Kubernetes and customized)
+ to provide network connectivity and security features for pod workloads. Considering the enormity and dynamism of Kubernetes
+ workloads in a cluster, Flow Visibility helps in the management and configuration of Kubernetes constructs such as Network
+ Policy, Services, Pods etc., and there by enhance the performance and security aspects of pod workloads.
+ 
+##Flow Exporter feature
+
+ In Antrea, basic building block to realize network flow visibility is the **Flow Exporter feature**. Flow exporter operates
+ at Antrea Agent; it builds and maintains a connection store by polling and dumping flows from conntrack module periodically.
+ The connections from connection store are exported to a flow collector using IPFIX protocol, and for this purpose we
+ use [go-ipfix](https://github.com/vmware/go-ipfix).
+ 
+###Configuration
+
+ To enable Flow Exporter feature at Antrea Agent, two config parameters has to be set in Antrea Agent configmap as shown 
+ below:
+```
+# Provide flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp. This also enables flow exporter that sends IPFIX
+# flow records of conntrack flows on OVS bridge. If no L4 transport proto is given, we consider tcp as default.
+# Defaults to "".
+#flowCollectorAddr: ""
+
+# Provide flow exporter poll and export intervals in format "0s:0s". This determines how often flow exporter polls connections
+# in conntrack module and exports IPFIX flow records that are built from connection store.
+# Flow export interval should be a multiple of flow poll interval.
+# Flow poll interval value should be in range [1s, ExportInterval(s)).
+# Flow export interval value should be in range (PollInterval(s), 600s].
+# Defaults to "5s:60s". Follow the time units of duration.
+#flowPollAndFlowExportIntervals: ""
+``` 
```

Comment:
for the configuration snipper, use the same format as this one (https://github.com/vmware-tanzu/antrea/blob/master/docs/feature-gates.md) for clarity, and remember the `yaml` tag

also show the feature gates part of the config

also merge the examples below with the snippet